tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLanguages.java

```diff
@@ -1372,171 +1372,187 @@ private String getLanguageName(String languageCode) {
-                                    "Coverage Goals",
-                                    null
-                                    // "<p>" +
-                                    // "The following show default coverage goals for larger
-                                    // organizations. " +
-                                    // "<i>[n/a]</i> shows where there is no specific value for a
-                                    // given organization, " +
-                                    // "while <i>(...)</i> indicates that the goal is inherited from
-                                    // the parent. " +
-                                    // "A * is added if the goal differs from the parent locale's
-                                    // goal. " +
-                                    // "For information on what these goals mean (comprehensive,
-                                    // modern, moderate,...), see the LDML specification "
-                                    // +
-                                    // "<a
-                                    // href='http://www.unicode.org/reports/tr35/#Coverage_Levels'>Appendix M: Coverage Levels</a>. " +
-                                    // +
-                                    // "</p>"
-                                    ,
-                                    null));
-
-            TablePrinter tablePrinter =
-                    new TablePrinter()
-                            // tablePrinter.setSortPriorities(0,4)
-                            .addColumn("Language", "class='source'", null, "class='source'", false)
-                            .setSortPriority(0)
-                            .setBreakSpans(false)
+            try (PrintWriter pw2 =
+                            new PrintWriter(
+                                    new FormattedFileWriter(
+                                            null,
+                                            "Coverage Goals",
+                                            null,
+                                            SUPPLEMENTAL_INDEX_ANCHORS));
+                    PrintWriter coverageGoalsTsv =
+                            FileUtilities.openUTF8Writer(
+                                    CLDRPaths.CHART_DIRECTORY + "tsv/", "coverage_goals.tsv"); ) {
+
+                TablePrinter tablePrinter =
+                        new TablePrinter()
+                                // tablePrinter.setSortPriorities(0,4)
+                                .addColumn(
+                                        "Language", "class='source'", null, "class='source'", false)
+                                .setSortPriority(0)
+                                .setBreakSpans(false)
+                                .addColumn(
+                                        "Code",
+                                        "class='source'",
+                                        "<a href=\"http://www.unicode.org/cldr/data/common/main/{0}.xml\">{0}</a>",
```

Comment:
```suggestion
                                        "<a href=\"https://www.unicode.org/cldr/data/common/main/{0}.xml\">{0}</a>",
```

1. http should be https
2. should use a CLDRURLS constant anyway
3. this URL looks wrong and doesn't seem to work. were you wanting something such as <https://github.com/unicode-org/cldr/blob/main/common/main/fr.xml> etc?